### PR TITLE
Add HTTPCodeServlet which allows for customizing the response code.

### PR DESCRIPTION
**Pull request by rhusar**

### CODE REVIEW

**Comment by rhusar** [comment]
@honza-kasik I am not a fan of the naming very much - do you mind if we change that? For example, the context rename to `/http-status-code` and parameter to `?code=200`?

File: clusterbench-ee10-ear/src/test/java/org/jboss/test/clusterbench/ear/HTTPCodeServletIT.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+                String responseBody = EntityUtils.toString(response.getEntity());
+                Assert.assertTrue(responseBody.contains("HTTP Code was: 503"));
+                Assert.assertTrue(responseBody.contains("JVM route: " + DebugServletIT.JBOSS_NODE_NAME));
+                Assert.assertTrue(responseBody.contains("Session isNew: false"));
+            }
+        }
+    }
+
+}
```
Comment (honza-kasik) [inline clusterbench-ee10-ear/src/test/java/org/jboss/test/clusterbench/ear/HTTPCodeServletIT.java:51]: Is missing newline at the end of the file OK?

**Review by honza-kasik — COMMENTED** [review]
Thanks! LGTM, just one question. +1 to naming change - is more self-explanatory.

File: clusterbench-ee10-ear/src/test/java/org/jboss/test/clusterbench/ear/HTTPCodeServletIT.java (hunk 1 above)
Comment (rhusar) [reply clusterbench-ee10-ear/src/test/java/org/jboss/test/clusterbench/ear/HTTPCodeServletIT.java]: It's ok that checkstyle doesn't complain, but I prefer to add it, fixed. Thanks.

**Comment by rhusar** [comment]
> @honza-kasik Updated with changes, please re-review. Thanks!

@honza-kasik Just a reminder ^^^

**Review by honza-kasik — APPROVED** [review]
Thanks, LGTM

**Pull request merged**